### ISSUE TITLE
cast streaming of the card gives black/blank panel

### ISSUE THREAD

**Issue by jurgenweber** [Makin-Things/bom-radar-card#19]
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser
- [x] I installed using HACS (leave unchecked for manual installs)

**Release with the issue:**

v1.6.0

**Last working release (if known):**

**Browser and Operating System:**
Chrome/OSX

**Card Config (click Show Code Editor and copy paste your config, remove a few of the least significant digits of your lat/lon to avoid posting you exact home address):**

```
  - title: Weather
    path: weather
    panel: true
    theme: dark
    cards:
      - type: horizontal-stack
        title: weather
        cards:
          - type: weather-forecast
            name: hourly
            entity: weather.kariong_hourly
          - type: weather-forecast
            name: daily
            entity: weather.kariong
          - type: 'custom:bom-radar-card'
            frame_count: 10
            center_latitude: -33.430999
            center_longitude: 151.287258
            show_marker: true
            show_range: true
            zoom_level: 10
```

**Description of problem:**

When I cast this onto my Chrome Cast/TV I only get a black/blank pane. I have given you the full config of the lovelace view that I cast. The weather cards are handled using the bureau_of_meteorology custom component.

**Additional information:**
Here is also the script I use to cast

```
cast_weather_living_tv:
  alias: "Show current weather on the living room tv"
  sequence:
    - data:
        dashboard_path: lovelace-cast
        entity_id: media_player.living_tv
        view_path: weather
      service: cast.show_lovelace_view
```

Love your work! I was sad when the old integration was removed due to scraping but now that I have found this, amazing.

**Comment by Makin-Things** [comment]
Have you been able to get casting to work for other dashboards? I have a google cast chrome ultra and I can't get anything to display on it :(

**Comment by jurgenweber** [comment]
Yeah! No Problems, I have my calendar and the weather rotating in a while loop. Its just this card that turns up blank.

**Comment by Makin-Things** [comment]
I finally got my Chromecast to work. HA Cast appears to not work with the majority of custom cards (there are community posts about this, but no obvious solutions). I think HA Cast itself needs more work. I'll do a bit more research but I doubt I will be able to make it work.

**Comment by DavidFW1960** [comment]
My BOM weather card is blank also. It USED to work but they broke the cast functionality badly when they 'fixed' it.

**Comment by Makin-Things** [comment]
@DavidFW1960 I have just figured out how to fix the bom-weather-card's ability to cast. The change will be in the rather large PR I submit once I have finished testing.
There are other problems with the radar card that with being able to get logs from the chromecast I doubt I will be able to solve. I suspect something to do with CORS, but yeah, no logs == no ability to debug :(

**Comment by burnsy180** [comment]
> @DavidFW1960 I have just figured out how to fix the bom-weather-card's ability to cast. The change will be in the rather large PR I submit once I have finished testing. There are other problems with the radar card that with being able to get logs from the chromecast I doubt I will be able to solve. I suspect something to do with CORS, but yeah, no logs == no ability to debug :(

Hey mate, I'm having the same issue as those above (at least I was yesterday). Upon casting to a Nest Hub using `cast.show_lovelace_view` I was getting a blank area where the bom-radar-card is supposed to be.
Having said that, tonight I tried it again and i'm getting the `Custom element doesn't exist: bom-radar-card` error and the yaml code in it's place.
I installed via HACS and have added the /local/community/ prefixes to new paths as others have suggested. ie new resource entry: /local/community/bom-radar-card/bom-radar-card.js

I have tried various other custom cards and they all seem to work when casting. I LOVE this card and am dying to use it as a popup when you click on my normal weather card (works fine on browser and mobile app). 

I've tried to get logs from the nest hub but I can't seem to find out how. Also all HA logs don't seem to mention this error.

Did you find a solution or have any suggestions to get this working? Or did anyone else?
